tiny/src/conn.rs

```diff
@@ -239,6 +239,7 @@ fn handle_irc_msg(ui: &dyn UI, client: &Client, msg: wire::Msg) {
                 if ui.user_tab_exists(serv, nick) {
                     ui.add_nick(nick, ts, &MsgTarget::User { serv, nick });
                 }
+                ui.set_tab_style(TabStyle::JoinOrPart, &MsgTarget::Chan { serv, chan: &chan })
```

Comment:
I think this should be in the "else" branch above. Currently we highlight the channel tab even when we join the channel (in the `new_chan_tab` part above), which doesn't make sense. We only want to highlight it when others are joining/leaving.

Reply:
It is inside the `else` block. Maybe I'm not understanding? Do you mean after line 240?

Reply:
Ah, you are right. Sorry for the noise.